dali/operators/decoder/image_decoder.cc

```diff
@@ -24,79 +24,110 @@ DALI_SCHEMA(ImageDecoderAttr)
       8*1024*1024)  // based on ImageNet heuristics (8MB)
   .AddOptionalArg("affine",
-      R"code(**`mixed` backend only** If internal threads should be affined to CPU cores)code",
+      R"code(Applies **only** to them ``mixed`` backend type.
+
+If set to True, each thread in the internal thread pool will be tied to a specific CPU core.
+Otherwise, the threads can be reassigned to any CPU core by the operating system.)code",
       true)
   .AddOptionalArg("split_stages",
-      R"code(**`mixed` backend only** Split into separated CPU stage and GPU stage operators)code",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+If True, the operator will be split into two sub-stages: a CPU and GPU one.)code",
       false)
   .AddOptionalArg("use_chunk_allocator",
-      R"code(**Experimental, `mixed` backend only** Use chunk pinned memory allocator, allocating chunk of size
-`batch_size*prefetch_queue_depth` during the construction and suballocate them
-in runtime. Ignored when `split_stages` is false.)code",
+      R"code(**Experimental**, applies **only** to the ``mixed`` backend type.
+
+Uses the chunk pinned memory allocator and allocates a chunk of the
+``batch_size * prefetch_queue_depth`` size during the construction and suballocates
+them at runtime. When ``split_stages`` is false, this argument is ignored.)code",
       false)
   .AddOptionalArg("use_fast_idct",
-      R"code(Enables fast IDCT in CPU based decompressor when GPU implementation cannot handle given image.
-According to libjpeg-turbo documentation, decompression performance is improved by 4-14% with very little
-loss in quality.)code",
+      R"code(Enables fast IDCT in the libjpeg-turbo based CPU decoder, used when ``device`` is set
+to "cpu" or when the it is set to "mixed" but the particular image can not be handled by
+the GPU implementation.
+
+According to the libjpeg-turbo documentation, decompression performance is improved by up to 14%
+with a low reduction in quality.)code",
       false)
   .AddOptionalArg("memory_stats",
-      R"code(**`mixed` backend only** Print debug information about nvJPEG allocations.
-The information about the largest allocation might be useful to determine suitable values for
-`device_memory_padding` and `host_memory_padding` for a given dataset.
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+Prints debug information about nvJPEG allocations. The information about the largest
+allocation might be useful to determine suitable values for ``device_memory_padding`` and
+``host_memory_padding`` for a dataset.
 
-Note: The statistics are global for the whole process (and not per operator instance) and include
-the allocations made during construction (when the padding hints are non-zero).)code",
+.. note::
+  The statistics are global for the entire process, not per operator instance, and include
+  the allocations made during construction when the padding hints are non-zero.
```

Comment:
```suggestion
  the allocations made during construction if the padding hints are non-zero.
```
Removed parenthesis = changed meaning :(

Reply:
Done